src/cmd/services/m3query/config/config.go

```diff
@@ -193,6 +193,14 @@ type Configuration struct {
 
 	// Debug configuration.
 	Debug config.DebugConfiguration `yaml:"debug"`
+
+	ProcessDelay *ProcessDelayConfiguration `yaml:"processDelay"`
+}
+
+// ProcessDelayConfiguration configures the ProcessDelay.
+type ProcessDelayConfiguration struct {
```

Comment:
Does this still get used?

Reply:
whoops. this was for testing. i'll remove.